### ISSUE TITLE
Provide support for retention policies

### ISSUE THREAD

**Issue by rolandoldengarm** [SubPointSolutions/spmeta2#1075]
As far as I know, SPMeta2 does not support deployment of retention policies to e.g. content types / lists. That would be awesome 👍

**Comment by SubPointSupport** [comment]
No, it does not yet. We'll see how it is supported by both CSOM/SSOM.

**Comment by andreasblueher** [comment]
I found these two links while trying to figure out if this is something I might want to implement

http://tech.bool.se/programmatically-create-setup-use-custom-site-policy/

http://www.dragan-panjkov.com/archive/2013/10/27/configuring-site-policy-in-sharepoint-2013-using-server-code.aspx

**Comment by SubPointSupport** [comment]
Thanks!
